back-end/apps/api/src/users/dtos/version-check-response.dto.ts

```diff
@@ -8,4 +8,42 @@ export class VersionCheckResponseDto {
   })
   @Expose()
   success: boolean;
+
+  @ApiProperty({
+    description: 'The latest supported frontend version (from server configuration)',
+    example: '1.2.0',
+    nullable: true,
+  })
+  @Expose()
+  latestSupportedVersion: string | null;
```

Comment:
Do the version strings need to be nullable? What does null mean other the frontend?

Reply:
It won't be null, cause it is taken from the .env 
made it required, so it won't be null